amethyst_gltf/src/systems.rs

```diff
@@ -0,0 +1,406 @@
+use assets::{AssetStorage, Handle, Loader};
+use core::transform::*;
+use renderer::{Material, MaterialDefaults, Mesh, Texture};
+use specs::{Entities, Entity, Fetch, FetchMut, Join, System, WriteStorage};
+use specs::common::Errors;
+
+use {GltfMaterial, GltfPrimitive, GltfSceneAsset};
+
+/// A GLTF scene loader, will transform `Handle<GltfSceneAsset>` into full entity hierarchies.
+///
+/// Will also do the asset storage processing for `GltfSceneAsset`.s
+pub struct GltfSceneLoaderSystem;
+
+impl GltfSceneLoaderSystem {
+    pub fn new() -> Self {
+        Self {}
```

Comment:
Meh, I don't like this syntax. And I guess we don't need this constructor, do we?

Reply:
Remnant from when it had fields